### ISSUE TITLE
Move "Choose a Role" dialog to below the user list

### ISSUE THREAD

**Issue by danielbachhuber** [goldenapples/Co-Authors-Plus-Roles#13]
... because it's secondary to the act of choosing an author.

It should probably have some default option of no role too.

**Comment by goldenapples** [comment]
Agreed. There was a suggestion to have a per-user default as well, to the role which that user has been assigned the most often.
